torchci/rockset/commons/__sql/flaky_workflows_jobs.sql

```diff
@@ -0,0 +1,46 @@
+-- Find workflows that succeeded once they were retried
+with repeats as (
+    select
+        j.run_id,
+        j.name
+    from
+        workflow_job j
+    where
+        j._event_time > CURRENT_TIMESTAMP() - DAYS(7)
+        and j.run_attempt > 1
+        and j.conclusion = 'success'
+) -- When a first time contributor submits a PR, their workflow starts at "attempt 2" so we look for workflows that were repeated above and actually have a first attempt
+select
+    job.head_sha as sha,
+    CONCAT(w.name, ' / ', job.name) as jobName,
+    job.id,
+    job.conclusion,
+    job.html_url as htmlUrl,
+    CONCAT(
+        'https://ossci-raw-job-status.s3.amazonaws.com/log/',
+        CAST(job.id as string)
+    ) as logUrl,
+    DATE_DIFF(
+        'SECOND',
+        PARSE_TIMESTAMP_ISO8601(job.started_at),
+        PARSE_TIMESTAMP_ISO8601(job.completed_at)
+    ) as durationS,
+    w.repository.full_name as repo,
+    job.torchci_classification.line as failureLine,
+    job.torchci_classification.captures as failureCaptures,
+    job.torchci_classification.line_num as failureLineNumber,
+from
+    workflow_job job
+    inner join workflow_run w on w.id = job.run_id
+    inner join repeats on repeats.run_id = job.run_id and repeats.name = job.name
+where
+    w.repository.full_name = :repo
+    and job._event_time >= PARSE_DATETIME_ISO8601(:startTime)
+    and job._event_time < PARSE_DATETIME_ISO8601(:stopTime)
+    and job.run_attempt = 1
+    and w.head_branch = :branch
+    and (
```

Comment:
Just curious, does `job.conclusion != 'success' and job.conclusion != 'null'` work here or is there other values to consider?  It doesn't make the logic simpler or shorter much I guess, so this is fine as long as this is consistent across all query lambdas querying flaky jobs

Reply:
According to https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_job, theres also neutral, action required, and stale.  I think skipped might also be one but I'm not entirely sure